sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/TrouterUtils.java

```diff
@@ -0,0 +1,351 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.chat.signaling;
+
+import com.azure.android.communication.chat.signaling.properties.ChatEventId;
+import com.azure.android.core.logging.ClientLogger;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+class TrouterUtils {
```

Comment:
If payload creation is the only thing this class does, consider rename it to what it does, e.g. ResponseParser or ResponseConverter